### PR TITLE
fix: Container + List + Modal + Statbox + Tabs reskinning

### LINKED ISSUE #3180: [Enhancement] Tabs reskinning
https://www.notion.so/appsmith/Tabs-42a6b3e57fbe450caebb0a4dfbb86ca7

Sometimes we want to have a custom styles of containers in the tabs of the tab widgets. An option to add custom style to tabs like background color would be great.

### LINKED ISSUE #10982: [Enhancement]: Modal widget reskinning
https://www.notion.so/appsmith/Modal-aceb586c6313452f99e54a6f4cc7b329

### LINKED ISSUE #10993: [Enhancement]: Stats box widget reskinning

### LINKED ISSUE #10999: [Enhancement]: List widget reskinning
https://www.notion.so/appsmith/List-4ef5910007254260b3fdb7c92b79594f

### PR DESCRIPTION
**Pull request by jsartisan**
## Description

> Please include a summary of the changes and which issue has been fixed. Please also include relevant motivation
> and context. List any dependencies that are required for this change.

Fixes #10982
Fixes #10999
Fixes #3180
Fixes #10993

> if no issue exists, please create an issue and ask the maintainers about this first

## Type of change

> Please delete options that are not relevant.

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- This change requires a documentation update

## How Has This Been Tested?

> Please describe the tests that you ran to verify your changes. Provide instructions, so we can reproduce.
> Please also list any relevant details for your test configuration.

- Test A
- Test B

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by laveena-en** [comment]
@jsartisan Could you please change the tab labels not to be bold? 
image

File: app/client/src/widgets/TabsWidget/widget/index.tsx [hunk 1]
```diff
@@ -372,8 +372,49 @@ class TabsWidget extends BaseWidget<
   static getPropertyPaneStyleConfig() {
     return [
       {
-        sectionName: "Border and Shadow",
+        sectionName: "Colors, Borders and Shadows",
         children: [
+          {
+            propertyName: "accentColor",
+            helpText: "Sets the checked state color of the checkbox",
```
Comment (dhruvikn) [inline app/client/src/widgets/TabsWidget/widget/index.tsx:379]: Seems like we missed changing the helpText. Can we please add a relevant message like
`Sets the color for the selected tab`

**Review by dhruvikn — CHANGES_REQUESTED** [review]

**Review by dhruvikn — APPROVED** [review]

**Pull request merged**